ts/src/binance.ts

```diff
@@ -5833,14 +5833,6 @@ export default class binance extends Exchange {
                 uppercaseType = market['contract'] ? 'TAKE_PROFIT' : 'TAKE_PROFIT_LIMIT';
             }
         }
-        if ((marketType === 'spot') || (marketType === 'margin')) {
```

Comment:
yeah we have to be careful with this, swap does not support full 
https://binance-docs.github.io/apidocs/futures/en/#new-order-trade